packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -173,6 +181,23 @@ export default function SubmitArticleModal({
     );
   };
 
+  useEffect(() => {
```

Comment:
Why you have to introduce it here?

Reply:
We have previously thought it was wrong as it was coming from a concluded experiment. After checking BE, it was then getting updated. Now that we have introduced the user state relevant to the eligibility, we will check from there. We should revert this change and have the same process but utilize the said property.

Reply:
We've updated to keep this here to avoid missing the tracking event as we open this modal from different actions.